multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/security/AuthorizationChecker.java

```diff
@@ -115,27 +116,26 @@ private boolean hasPermissions(CloudControllerClient client, SpaceWithUser space
         if (isSpaceDeveloperUsingCache(client, spaceWithUser)) {
             return true;
         }
-        UserRole userRole = refreshUserRole(client, spaceWithUser);
-        if (userRole.hasSpaceRole(UserRole.SpaceRole.SPACE_DEVELOPER)) {
+        List<UserRole> userRoles = refreshUserRoles(client, spaceWithUser);
+        if (userRoles.contains(UserRole.SPACE_DEVELOPER)) {
             return true;
         }
-        return readOnly
-            && (userRole.hasSpaceRole(UserRole.SpaceRole.SPACE_AUDITOR) || userRole.hasSpaceRole(UserRole.SpaceRole.SPACE_MANAGER));
+        return readOnly && (userRoles.contains(UserRole.SPACE_AUDITOR) || userRoles.contains(UserRole.SPACE_MANAGER));
```

Comment:
We can change the condition, because now if user id is has some space role it could read:
(userRoles.contains(UserRole.SPACE_AUDITOR) || userRoles.contains(UserRole.SPACE_MANAGER))
with
!userRoles.isEmpty()

Reply:
What should happen if the user has organization roles, but not space roles?